cmd/root.go

```diff
@@ -204,6 +204,7 @@ func initConfig() {
 		viper.AddConfigPath(path.Join(usr.HomeDir, ".exoscale"))
 		viper.AddConfigPath(usr.HomeDir)
 		viper.AddConfigPath(".")
+		gConfigFilePath = path.Join(gConfigFolder, fmt.Sprintf("%s.toml", defaultConfigFileName))
```

Comment:
it looks like #63 material.

Reply:
@pierre-emmanuelJ is this still required?

Reply:
It's to display more explicite error message in runstatus

Reply:
did you read #63 changes?

Reply:
thanks you did that in root.go L207